From 5d0c9d2217dbe369daffb8a92d7b5e7d7d34d566 Mon Sep 17 00:00:00 2001
From: Richard Marian Thomaiyar <richard.marian.thomaiyar@linux.intel.com>
Date: Sat, 2 Mar 2019 20:08:32 +0530
Subject: [PATCH] Fix: User commands require channel layer lib

As channel layer is separated out from user layer lib, it
has to be manually included in libusercommands, as user
command handlers use channel layer API's

Tested-by:
1. Made sure that libusercommands are loaded on it's own
without any undefined symbol error.
2. ipmitool user list 1 works on host interface

Change-Id: I6652ad248e01afc1349e3a9612754dbdb84b96ad
Signed-off-by: Richard Marian Thomaiyar <richard.marian.thomaiyar@linux.intel.com>
---
 Makefile.am | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Makefile.am b/Makefile.am
index bb7bdbf..4e9101e 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -120,7 +120,8 @@ libipmi20_la_CXXFLAGS = $(COMMON_CXX)
 providers_LTLIBRARIES += libusercmds.la
 libusercmds_la_LIBADD = \
 	libipmid/libipmid.la \
-	user_channel/libuserlayer.la
+	user_channel/libuserlayer.la \
+	user_channel/libchannellayer.la
 libusercmds_la_SOURCES = \
 	user_channel/usercommands.cpp
 libusercmds_la_LDFLAGS = \
-- 
2.7.4

